_security-plugin/index.md

```diff
@@ -8,20 +8,62 @@ redirect_from:
+
+Another notable characteristic of access control in OpenSearch is the ability to assign user access within the domain through levels of increasing granularity. Fine grained access control (FGAC) means that a role can control permissions for users at not only the cluster level, but the index level, the document level, and even the field level. For example, a role may provide a user access to certain cluster-level permissions but at the same time prevent the user from accessing a given group of indexes. Likewise, that role may grant access to certain types of documents but not others, or even include access to specific fields within a document but exclude access to other sensitive fields. Field masking further extends the fine grained control of data access by providing options to mask certain types of data, such as a list of emails, which can still be aggregated but not made viewable to a role.
+
+To learn more about this feature, see the [Access control](https://opensearch.org/docs/latest/security-plugin/access-control/index/) section of security documentation.
+
+### Audit logging and compliance
+
+Finally, audit logging and compliance refer to mechanisms that allow for tracking and analysis of activity within the cluster. This becomes particulalry important after events where the cluster suffers unintended data exposure. These logging features enable you to oversee changes made anywhere in the cluster – such as alterations to a domain or the modification of parameters – and monitor access patterns and requests of all types, whether valid or invalid.
+
+How OpenSearch archives logging is configurable at many levels of detail, and there are a numbner of options for where those logs are stored. Compliance features also ensure that all data is available if and when compliance auditing is required. In this case, the logging can be automated to focus on data especially pertinent to those compliance requirements.
+
+See the [Audit logs](https://opensearch.org/docs/latest/security-plugin/audit-logs/index/) section of security documentation to read more about this feature.
+
+## Other features and functionality
+
+Security includes other features that make working in OpenSearch easier and more secure.
+
+One such feature is OpenSearch Dashboards multi-tenancy. Tenants provide a way to safely share Dashboards user configurations with other Dashboards users, including visualizations, index patterns, and other Dashboards objects. You can also set up roles that control access to your tenants.
```

Comment:
```suggestion
One such feature is OpenSearch Dashboards multi-tenancy. Tenants provide a way to separate Dashboards user configurations with other Dashboards users, including visualizations, index patterns, and other Dashboards objects. You can also set up roles that control access to your tenants.
```

The current feature doesn't really provide a safe 'security' boundary, but instead its an partitioning mechanism.  In other systems this is called 'workspaces' which is I think more appropriate, but its already named :D 

When I think about a tenant models, I think about an apartment complex.  If you placed a chair in one apartment, it isn't assumed to be accessible in another apartment, things outside the apartment complex aren't generally considered.

However, the model is dashboards is slight different in that the item of interest are the fixtures within (index patterns, dashboards) but its also of interest is the view outside the apartment window (data) - which is common between all apartments.

Maybe @jimishs has thoughts on how we frame this?

Reply:
Thanks for the update, this looks good to me